app/Http/Controllers/Users/AgencyController.php

```diff
@@ -77,10 +77,10 @@ public function logoUpload()
             'logo.file_extension' => __('validation.agency.logo.file_extension'),
         ]);
 
-        \File::delete(public_path('assets/imgs/'.Option::get('agency_logo_path')));
+        \File::delete(storage_path('app/public/assets/imgs'.Option::get('agency_logo_path')));
```

Comment:
We need a slash here. Without it, we will get an invalid file's absolute path, so the old logo is not deleted from the storage.

```suggestion
        \File::delete(storage_path('app/public/assets/imgs/'.Option::get('agency_logo_path')));
```